### ISSUE TITLE
[SUGGESTION] - Show CG values in EFB

### ISSUE THREAD

**Issue by falcon71** [invernyx/md11-bugs#1237]
### Is your feature request related to a problem? Please describe.

If no WBS is installed, then there is currently no way to figure out the ZFWCG and TOCG for entry in the WEIGHT INIT page in the FMC.

### Describe the solution you'd like

I'd like to be able to read the ZFWCG and TOCG in the payload tab in the EFB

### Describe alternatives you've considered

There is currently a bug (#904) that allows reading the current actual CG from the ENG page without having them entered in the FMC.
Installing the WBS is another workaround.

### Additional context

-

**Comment by GenericNerd** [comment]
We won't be implementing this as there is a solution already provided, the WBS system. If you do not have it, you will have to find another way to get it, such as the simBrief flight plan. The EFB will not be providing this for you.

**Comment by KJA1582** [comment]
SB does not provide CG values

**Comment by GenericNerd** [comment]
> SB does not provide CG values

Never mind, you are correct. Either way, you can get the CG from the WBS system.

**Comment by KJA1582** [comment]
> > SB does not provide CG values
> 
> Never mind, you are correct. Either way, you can get the CG from the WBS system.

True, but as it is an optioan that we can disable, it makes sense to have the EFB (which gives us a pseudo loadhseet) to include this.
IIRC, IRL you would compare the Loadsheet value with the WBS values and only use the WBS if it is within an error margin to the Loadsheet value

**Comment by joon44** [comment]
We only need the ZFWCG in the EFB. TOCG can be obtained from the ENG on FUEL SD pages.
